opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1235,11 +1244,12 @@ namespace Opm {
     {
 
         DeferredLogger local_deferredLogger;
-        if (this->glift_debug) {
+/*        if (this->glift_debug) {
             const std::string msg = fmt::format(
                 "assemble() : iteration {}" , iterationIdx);
             this->gliftDebug(msg, local_deferredLogger);
         }
+*/
```

Comment:
Just out of curiosity: Why do we disable this debugging facility?

Reply:
nice catch. that was supposed to be a temporary while refactoring but things got a little muddy. I had some follow-ups for this but they require some somewhat unrelated reorganizations. I now realize I need to do it here tough, so there's an additional commit in the PR now, moving some things around.